scripts/main.js

```diff
@@ -46,10 +46,10 @@ function openOverlayWithImage(imgUrl) {
   overlayElement.classList.remove('hidden');
 }
 
-function addThumbnailClickListeners() {
+function addClickListeners(container, filter) {
```

Comment:
Perhaps, consider using a more specific function name. Such as...addListenersToElements.

Reply:
I like it@

Reply:
I think I'll go with `addListeners`. More generic, because that makes sense.